app/actions/configurations.py

```diff
@@ -0,0 +1,43 @@
+import pydantic
+
+from .core import AuthActionConfiguration, PullActionConfiguration, ExecutableActionMixin
+from ..services.errors import ConfigurationNotFound
+from ..services.utils import find_config_for_action
+
+
+class AuthenticateConfig(AuthActionConfiguration, ExecutableActionMixin):
+    username: str
+    password: pydantic.SecretStr = pydantic.Field(..., format="password")
+    grant_type: str = "password"
+
+
+class PullObservationsConfig(PullActionConfiguration):
+    observations_per_request: int = 200
```

Comment:
We may not want to show this setting in the UI or let users change it. In other integrations I [put it in the integration settings file reading it from an env var](https://github.com/PADAS/gundi-integration-ats/blob/516ee17fafda7f0a3de5e677018025210de3b938/app/settings/integration.py#L6) so we can change it if needed.